packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -11,6 +11,17 @@ import { isKeyboardActive } from '@vaadin/component-base/src/focus-utils.js';
 import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
```

Comment:
https://user-images.githubusercontent.com/1222264/187194820-779294fe-0629-4bdd-b306-1c5e33068ca2.mp4

Seems that the example on the dev page now shows just one tooltip

Reply:
Right, this happens because of closing other tooltips when a tooltip is shown.
I think we can remove this logic for now, and only rely on listeners.

Reply:
Fixed by changing the logic to flush closing for tooltips that are already scheduled to be closed.
This does not prevent multiple tooltips from being shown simultaneously, as in the dev page.